Source/WebCore/rendering/mac/RenderThemeMac.mm

```diff
@@ -1744,6 +1744,19 @@ static void setFontFromControlSize(RenderStyle& style, NSControlSize controlSize
     return StringTruncator::centerTruncate(strToTruncate, width, font);
 }
 
+String RenderThemeMac::fileListName(const FileList* fileList, bool multipleFilesAllowed) const
```

Comment:
Can we refactor `fileListNameForWidth` to call `fileListName`? It seems there's a bit of duplication happening here with the existing method.